client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -102,6 +95,49 @@ export default function WPCheckoutOrderSummary() {
 	);
 }
 
+function CheckoutSummaryFeaturesList() {
+	const hasDomainsInCart = useHasDomainsInCart();
+	const domains = useDomainsInCart();
+	const hasPlanInCart = useHasPlanInCart();
+	const translate = useTranslate();
+
+	const supportText = hasPlanInCart
+		? translate( 'Email and live chat support' )
+		: translate( 'Email support' );
+
+	let refundText = translate( 'Money back guarantee' );
+	if ( hasDomainsInCart && ! hasPlanInCart ) {
+		refundText = translate( '4 day money back guarantee' );
+	} else if ( hasPlanInCart && ! hasDomainsInCart ) {
+		refundText = translate( '30 day money back guarantee' );
+	}
+
+	return (
+		<CheckoutSummaryFeaturesListUI>
+			{ hasDomainsInCart &&
+				domains.map( ( domain ) => {
+					return (
+						<CheckoutSummaryFeaturesListItem>
+							<WPCheckoutCheckIcon />
+							<strong>{ domain.wpcom_meta?.meta }</strong>
+							{ domain.wpcom_meta?.is_bundled && (
+								<BundledDomainFreeUI>{ translate( 'free with plan' ) }</BundledDomainFreeUI>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **33** times:
`translate( 'Free with Plan' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/checkout-dynamic-plan-features)